nautobot/ipam/templates/ipam/vlangroup.html

```diff
@@ -82,9 +82,17 @@ <h1>{{ object }}</h1>
                 <tr>
                     <td>VLANs</td>
                     <td>
-                        <a href="{% url 'ipam:vlan_list' %}?group={{ object.pk }}">{{ vlan_table.rows|length }}</a>
+                        <a href="{% url 'ipam:vlan_list' %}?group={{ object.pk }}">{{ vlan_table.rows|length|add:"-1" }}</a>
```

Comment:
Is this a fix for #490?

Reply:
Yes, actually that appears to be a fix for #490. I can take a look a bit more and submit this snippet for that.

Reply:
By take a look a bit more, I mean test this specific fix a bit more.

Reply:
Yeah actually it looks like more would be needed here, since if the allocated vlans are not contiguous, the number may be off by something other than 1.